src/pycparser/c_parser.py

```diff
@@ -123,7 +123,7 @@ def __init__(
         # saw: int name;
         # If 'name' is not a key in _scope_stack[n] then 'name' was not defined
         # in this scope at all.
-        self._scope_stack = [dict()]
+        self._scope_stack = [()]
```

Comment:
This change seems odd? meant to be "{}" for the literal element of the list?

Reply:
That is correct it should be {}